### PR TITLE
docs: adding handoff, root-credentials, and 2.0 fixes

### PR DESCRIPTION
**Pull request by johndietz**
- adding latest version of handoff screens
- adding root-credentials command and guidance
- adding connecting to kubernetes sections
- removing typos, lies, and mayhem

### CODE REVIEW

**Review by jarededwards — DISMISSED** [review]

**Comment by fharper** [comment]
The prefix for this PR would be `docs` since you are adding new documentation (most of this PR, the rest would have been a `fix`). I changed it for you, but wanted to share the reason. The `feat` prefix is for new feature for the documentation platform, like when we added the search with Typesense.

It's a [bit different for docs](https://github.com/kubefirst/docs/blob/main/CONTRIBUTING.md#commits), than coding.

If those changes are also for 2.0, which I think they are, [you need to duplicate your changes](https://github.com/kubefirst/docs/blob/main/CONTRIBUTING.md#versioning) in the `versioned_docs/version-2.0` folder as mentioned in Slack (adding here for tracking purpose).

File: docs/explore/certificates.md [hunk 1]
```diff
@@ -4,38 +4,12 @@ title: Certificates
 
 ## SSL Certificates
 
-Kubefirst takes advantage of a variety of security layers to make sure the platform is safe and secure for our users. One important and significant layer is the SSL certificates Kubefirst leverages to protect HTTPS requests. [Let's Encrypt](https://letsencrypt.org) is the certificate authority used to manage the certificates used for our ingressed services.
+In cloud versions of the kubefirst platform, we use [Cert Manager](https://cert-manager.io/) with a [Let's Encrypt](https://letsencrypt.org) clusterissuer for TLS encryption to all of our ingressed services. These certificates are requested automatically, and will auto-renew.
```
Comment (fharper) [inline docs/explore/certificates.md:7]: Cert Manager = cert-manager
clusterissuer = ClusterIssuer

I would change "all of our ingressed services" to "all our services on Ingress". I'm fined using a verb that doesn't exist, but that people still used. In that case, just update add a capital I to ingressed.

File: docs/explore/terraform.md [hunk 2]
```diff
@@ -8,9 +8,11 @@ Terraform is our infrastructure as code layer and we manage our Terraform workfl
 
 ### Automatic Plans With Atlantis
 
-Any merge request that includes a .tf file will prompt `atlantis` to wake up and run your Terraform plan. Atlantis will post the plan's result to your merge request as a comment within a minute or so.
+Any pull request that includes a .tf file will prompt `atlantis` to wake up and run your Terraform plan. Atlantis will post the plan's result to your pull request as a comment within a minute or so.
```
Comment (fharper) [inline docs/explore/terraform.md]: Would keep both since this page is used for GitHub & GitLab.

File: docs/explore/terraform.md [hunk 3]
```diff
@@ -8,9 +8,11 @@ Terraform is our infrastructure as code layer and we manage our Terraform workfl
...
 
-Review and eventually approve the merge request.
+Review and approve the pull request.
```
Comment (fharper) [inline docs/explore/terraform.md]: Would keep both since this page is used for GitHub & GitLab.

File: docs/explore/terraform.md [hunk 4]
```diff
@@ -8,9 +8,11 @@ Terraform is our infrastructure as code layer and we manage our Terraform workfl
...
+
+You can always run a new terraform plan by commenting directly in your pull request with the words `atlantis plan`.
```
Comment (fharper) [inline docs/explore/terraform.md]: terraform = Terraform

**Review by fharper — DISMISSED** [review]

**Review by echoboomer — APPROVED** [review]

**Pull request merged**